src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_NullCoalescingOperator.cs

```diff
@@ -114,6 +114,15 @@ public override BoundNode VisitNullCoalescingOperator(BoundNullCoalescingOperato
                 }
             }
 
+            // Optimize left ?? right to left.GetValueOrDefault() when left is T? and right is the default value of T
+            if (rewrittenLeft.Type.IsNullableType()
+                && rewrittenRight.IsDefaultValue()
+                && rewrittenRight.Type == rewrittenLeft.Type.GetNullableUnderlyingType()
+                && TryGetNullableMethod(rewrittenLeft.Syntax, rewrittenLeft.Type, SpecialMember.System_Nullable_T_GetValueOrDefault, out MethodSymbol getValueOrDefault))
+            {
+                return BoundCall.Synthesized(rewrittenLeft.Syntax, rewrittenLeft, getValueOrDefault);
```

Comment:
>return BoundCall.Synthesized(rewrittenLeft.Syntax, rewrittenLeft, getValueOrDefault); [](start = 16, length = 85)

Please make similar changes in VB compiler.

Reply:
Not done yet, I'm waiting for other changes to be approved first, so I can add similar tests, etc.

Reply:
I've added VB changes and tests.